railties/test/generators/db_system_change_generator_test.rb

```diff
@@ -90,6 +90,25 @@ class ChangeGeneratorTest < Rails::Generators::TestCase
             end
           end
 
+          test "change to trilogy" do
+            run_generator ["--to", "trilogy"]
+
+            assert_file("config/database.yml") do |content|
+              assert_match "adapter: trilogy", content
+              assert_match "database: tmp_production", content
+            end
+
+            assert_file("Gemfile") do |content|
+              assert_match "# Use trilogy as the database for Active Record", content
+              assert_match 'gem "trilogy", "~> 2.4"', content
+            end
+
+            assert_file("Dockerfile") do |content|
+              assert_match "build-essential git", content
+              assert_match "curl libvips", content
```

Comment:
Maybe we should add `assert_no_match "default-libmysql-client"`, etc? Otherwise if someone accidentally adds back the dependency this test will still pass, right?

Reply:
Make sense to me. I added checking `default-libmysqlclient-dev`. Thanks!

Reply:
@y-yagi Why not add one for `default-mysql-client` too?

Reply:
@zzak 
This is a test to prevent to addition of MySQL client packages when using Trilogy adapter. So I thought one package is enough.